crates/compiler/src/builtins.rs

```diff
@@ -951,18 +949,18 @@ fn mk_builtin_table() -> Vec<Builtin> {
     ]
 }
 
-pub fn make_builtin_labels() -> HashMap<Symbol, Name> {
+pub fn make_builtin_offsets() -> HashMap<Symbol, usize> {
```

Comment:
Nit, not really related to the changes this PR is making: this (and also `make_offsets_builtins`) could in theory calculated at compile time. In practice, not so much. Maybe with https://crates.io/crates/phf if we *really* cared, but I don't think we do. So: maybe stick these in `lazy_static` too?

Reply:
Thanks!  I cleaned this up a bunch, created a separate "Builtins" registry with a BuiltinId, etc. etc.